grammars/language-asciidoc.cson

```diff
@@ -1221,6 +1224,14 @@ repository:
         end: "^(\\1)$"
       }
     ]
+  "literal-paragraph":
+    patterns: [
+      {
+        name: "markup.block.literal.asciidoc"
+        begin: "^\\p{Blank}+(.*)$"
```

Comment:
I think this should be `.+` (instead of `.*`). There needs to be at least one non-whitespace character in the line.